### PR TITLE
Master payment payulatam payment integration nla

### PR DESCRIPTION
**Pull request by nla-odoo**
Added a new payment acquirer PayUlatam.

Task: https://www.odoo.com/web#id=1826303&action=327&model=project.task&view_type=form&menu_id=4720

Pad: https://pad.odoo.com/p/r.589bf0ced97b289b18d5965d8cfaec64

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/payment_payulatam/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    'category': 'Payment Acquirer',
+    'summary': 'Payment Acquirer: PayuLatam Implementation',
+    'description': """Payulatam payment acquirer""",
+    'depends': ['payment'],
+    'data': [
+        'views/payment_views.xml',
+        'views/payment_payulatam_templates.xml',
+        'data/payment_acquirer_data.xml',
+    ],
+}
```
Comment (mreficent) [inline addons/payment_payulatam/__manifest__.py]: I think you should add `'post_init_hook': 'create_missing_journal_for_acquirers',` like in the others `payment_x` modules

**Comment by mba-odoo** [comment]
LGTM

**Comment by jacobus101** [comment]
Is this gonna be in v12?

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**